style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a good practice to make all colors variable

Reply:
Done